src/components/side-navigation/NavigationHeader.tsx

```diff
@@ -1,3 +1,5 @@
+import styles from "./NavigationHeader.module.sass";
```

Comment:
Import from the main `SideNavigation.module.sass` stylesheet instead.
```suggestion
import styles from "./SideNavigation.module.sass";
```